superset/utils.py

```diff
@@ -881,4 +881,3 @@ def split_adhoc_filters_into_base_filters(fd):
         fd['having'] = ' AND '.join(['({})'.format(sql) for sql in sql_having_filters])
         fd['having_filters'] = simple_having_filters
         fd['filters'] = simple_where_filters
-        del fd['adhoc_filters']
```

Comment:
Note we can no longer remove the `adhoc_filters` field as we need to ensure that ad-hoc filters persist. Note currently by deleting this field the ad-hoc filters actually get rebuilt on the frontend by merging the legacy and ad-hoc filters.